backend/src/packages/journal-entries/journal-entry.controller.ts

```diff
@@ -132,6 +149,57 @@ class JournalEntryController extends BaseController {
       payload: await this.journalEntryService.findAll(),
     };
   }
+
+  /**
+   * @swagger
+   * /journal/{id}:
+   *   delete:
+   *     description: Delete a journal entry by ID
+   *     security:
+   *       - bearerAuth: []
+   *     parameters:
+   *       - in: path
+   *         name: id
+   *         description: ID of the journal entry to delete
+   *         required: true
+   *         schema:
+   *           type: integer
+   *     responses:
+   *       200:
+   *         description: Successfully deleted the journal entry
+   *         content:
+   *           application/json:
+   *             schema:
+   *               type: object
+   *               properties:
+   *                 isDeleted:
+   *                   type: boolean
+   *                   description: Is successfully deleted
+   *       404:
+   *         description: Incorrect user credentials
+   *         content:
+   *           application/json:
+   *             schema:
+   *               $ref: '#/components/schemas/Error'
+   *             example:
+   *               message: "Incorrect credentials."
+   *               errorType: "AUTHORIZATION"
+   */
+
+  private async delete(
+    options: APIHandlerOptions<{
+      user: UserAuthResponseDto;
+      params: JournalDeleteParameter;
+    }>,
+  ): Promise<APIHandlerResponse> {
+    return {
```

Comment:
Check it return value is true, otherwise 404

Reply:
@v0ldemar01 it should be done in service?

Reply:
No, in controller

Reply:
done